docs/reference/setup/install/docker.asciidoc

```diff
@@ -202,7 +203,17 @@ endif::[]
 To stop the cluster, type `docker-compose down`. Data volumes will persist, so it's possible to start the cluster again with the same data using `docker-compose up`.
 To destroy the cluster **and the data volumes** just type `docker-compose down -v`.
 
-===== Inspect status of cluster:
```

Comment:
This patch is submitted against an older version of the Docker docs (branch `5.1`). Unless it's a very branch specific change, the prefer approach it to submit patches against `master`, in which case Elastic will backport to the needed branches.